src/openapi/peridio-device-openapi.yaml

```diff
@@ -153,6 +153,12 @@ paths:
             description: The PRN of the currently active release on the device.
             $ref: "peridio-admin-openapi.yaml#/components/schemas/prn"
           required: true
+        - in: header
+          name: peridio-release-version
+          schema:
+            description: The desired version of the release that the device wants to be updated to as part of the [dynamic release resolution](/reference/releases#using-the-peridio-release-version-header) process.
```

Comment:
Not super happy about this description. Lemme know if this sounds good @danielspofford

Reply:
i pushed an update, lmk